pkg/kubelet/images/image_manager.go

```diff
@@ -50,12 +59,26 @@ type imageManager struct {
 	puller imagePuller
 
 	podPullingTimeRecorder ImagePodPullingTimeRecorder
+
+	keyring *credentialprovider.DockerKeyring
+	// ensureSecretPulledImages - map of imageref (image digest) to successful secret pulled image details
```

Comment:
I am not sure if this comment adds any value. One can see the type of the map values here clearly.

Reply:
Yeah, on the other note, I felt extra detailing was good and hence did not delete the comment.